grobid-core/src/main/java/org/grobid/core/data/BiblioItem.java

```diff
@@ -1982,11 +1986,14 @@ public String toBibTeX(String id, GrobidAnalysisConfig config) {
 
             // dates
             if (normalized_publication_date != null) {
-                String isoDate = TEIFormatter.toISOString(normalized_publication_date);
+                String isoDate = Date.toISOString(normalized_publication_date);
                 if (isoDate != null) {
                     bibtex.add("  date = {" + isoDate + "}");
                 }
             }
+            if (publication_date != null) {
+                bibtex.add("  year = {" + publication_date + "}");
+            }
```

Comment:
I think `date` is preferred over `year` and if used, `year` should only consist of a numeric year.
I see two options here:

1) Put this in an else branch to the if before it and write it to the `date` field.
So, if a `normalized_publication_date` exists, use it as iso string, else if the `publication_date` exists use that.
2) Don't format the `normalized_publication_date` as ISO string but use the `year`, `month` and `day` fields instead. This would ensure that `year`, `month` and `day` are numeric and the (more detailed, with day ranges) `publication_date` could be put as a string in the `date` field. If both are given, it is up to the user to decide which to keep.